lib/components/base/ProjectCard.vue

```diff
@@ -83,147 +86,69 @@ import {
 import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime.js'
```

Comment:
not directly connected to this PR but...
we should probably add [this](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports) to the `tsconfig.json` to allow this type of import

else my webstorm screams at me...

Reply:
What does WebStorm say? Oddly, I have no problems with this in VS Code, but I think both WS and VSC use the same TS Language server...

Reply:
WebStorm just marks it as wrong and as a fix suggests adding `allowSyntheticDefaultImports` to the `tsconfig.json` with not much additional information.

![grafik](https://github.com/modrinth/omorphia/assets/81473300/52f5d584-cf54-4ac5-ac3a-a12a2de9dade)

Reply:
I've tried WebStorm and could reproduce, yeah. I think WS is just getting confused by require.js and CJS compatible wrapper? There's no such error in VS Code. But I don't think it hurts to add it. Opened #159 to fix this.